### ISSUE TITLE
Brick.Widgets.Table behaves weirdly if there are empty cells

### ISSUE THREAD

**Issue by pmiddend** [jtdaugherty/brick#369]
Due to the way brick's tables are constructed, if you have empty cells in the table, then those will not be drawn at all and the table will be skewed. For example, say you have this table:

```
| foo | bar |
|   1 |   2 |
|   x |     |
|   2 |   3 |
```

If you try to draw this using Brick's `Table` class, what results is:

```
| foo | bar |
|   1 |   2 |
|   x |   3 |
|   2 |     |
```

This is due to the fact that you `hBox` the columns next to each other and every column is just a `vBox` of its consitutents. However, one can just use `txt ""` as a cell widget, which has no vertical extent. It'll then be omitted in the table.

I'm not sure if this is a bug or can be fixed - or if it might just warrant a documentation entry.

**Comment by jtdaugherty** [comment]
Hi @pmiddend - thank you for reporting this! I just pushed a patch to fix this behavior in 55c13f7ff4e997d5195c9c2da3e38bcd9b29669b.

In testing this fixed, I was also reminded of another situation: tables do not behave well when they have a row or column with *all* empty cells since the table rendering algorithm needs some non-empty cell in each column and row to determine the sizes of the other cells. That isn't a great edge case and the rendering results won't be great, but I'd have to think harder about how that case could be handled. In any case, I added a cautionary note about that to the `table` documentation.

**Comment by jtdaugherty** [comment]
This fix is now released in 0.69.1.

**Comment by jtdaugherty** [comment]
@pmiddend FYI I just released 0.70 which has more improvements to how tables behave when some or all cells are empty. I suggest upgrading to 0.70 to get the benefit of the latest changes.

**Comment by pmiddend** [comment]
@jtdaugherty Thanks, will do! And thanks for the rapid reply
